.storybook/config.js

```diff
@@ -9,4 +10,10 @@ function loadStories() {
   reqInSrcTree.keys().forEach(filename => reqInSrcTree(filename));
 }
 
+addDecorator(
```

Comment:
Sets the name "Photon Components" instead of Storybooks at the top of the navbar